docs/src/components/banner/AppHeaderBanner.tsx

```diff
@@ -10,7 +10,7 @@ export default function AppHeaderBanner() {
       fontWeight="medium"
       sx={{
         color: '#fff',
-        p: 1.2,
+        p: '12px',
```

Comment:
I think that it would be great to have an eslint plugin to forbid this pattern. I think that only integer and half-integer make sense with the unitless API. It's effectively the same as https://stackoverflow.com/questions/54618144/tailwind-css-how-to-code-pixel-perfect-design.

Reply:
Added the idea to #31096.

Reply:
Agree. The reason to use these broken values is just pure eye-ball refinement, nothing too special about them. But yeah, let's do it!